docs/make.jl

```diff
@@ -1,15 +1,19 @@
-using Documenter, StaticArrays
+using Documenter
+using StaticArrays
 
-makedocs(
-         modules = [StaticArrays],
-         sitename = "StaticArrays.jl",
-         pages = [
-            "Home" => "index.md",
-            "API" => "pages/api.md",
-            "Quick Start" => "pages/quickstart.md",
-            ],
-        )
-
-deploydocs(
-    repo = "github.com/JuliaArrays/StaticArrays.jl.git",
+makedocs(;
+    modules = [StaticArrays],
+    format = Documenter.HTML(
+        canonical = "https://JuliaArrays.github.io/StaticArrays.jl/stable/",
+        assets = ["assets/favicon.ico"],
+    ),
+    pages = [
+        "Home" => "index.md",
+        "API" => "pages/api.md",
+        "Quick Start" => "pages/quickstart.md",
+        ],
+    repo = "https://github.com/JuliaArrays/StaticArrays.jl/blob/{commit}{path}#L{line}",
```

Comment:
This should not be necessary to specify for GitHub hosted packages.

Reply:
Sorry, I just copied from PkgTemplates.jl.
I'll remove it.